pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -151,6 +151,18 @@ func GetSubjectDigests(tr *v1beta1.TaskRun, logger *zap.SugaredLogger) []intoto.
 		}
 	}
 
+	sts := artifacts.ExtractIntotoSignableTargetFromResults(tr, logger)
+	for _, obj := range sts {
+		splits := strings.Split(obj.(*artifacts.StructuredSignable).Digest, ":")
+		logger.Warnf("Split %s %s", splits[0], splits[1])
```

Comment:
Why is this a warning?

Also, could you check the length of `splits` to ensure it is always 2? A warning if it's not 2 followed by a `continue` will handle some corner case we haven't thought of.